lib/pages/new_group/new_group.dart

```diff
@@ -68,6 +78,35 @@ class NewGroupController extends State<NewGroup> with ComparablePresentationCont
     });
   }
 
+  void listenCreateRoom() {
+    createRoomStreamController.streamController.stream.listen((event) {
+      Logs().d('NewGroupController::createRoom() - event: $event');
+      event.fold(
+        (failure) => {
+          Logs().e('NewGroupController::createRoom() - failure: $failure'),
+          createRoomStreamController.dispose(),
```

Comment:
you shouldn't call dispose here, if there are listeners listen for it, it will throw exception. you should dispose in `dispose() method`

Reply:
sure, it is not a good solution. `createRoomStreamController` is a global variable, it is risky